edgexfoundry/device-sdk-go
# feat!: updates for common config

BREAKING CHANGE: configuration file changed to remove common config settings

Signed-off-by: Elizabeth J Lee <elizabeth.j.lee@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) - addresses bootstrap changes
  <link to docs PR>

## Testing Instructions
1. run make build to build device simple
2. run the edgex stack in non-secure mode
3. run device simple with config provider
4. verify that the config exists in the config provider

## New Dependency Instructions (If applicable)
